<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGradientDialog</class>
 <widget class="QDialog" name="CGradientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="listWidget"/>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="addButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="object_viewer.qrc">
             <normaloff>:/icons/ic_nel_add_item.png</normaloff>:/icons/ic_nel_add_item.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="insButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="object_viewer.qrc">
             <normaloff>:/icons/ic_nel_insert_item.png</normaloff>:/icons/ic_nel_insert_item.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="object_viewer.qrc">
             <normaloff>:/icons/ic_nel_delete_item.png</normaloff>:/icons/ic_nel_delete_item.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="upButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="object_viewer.qrc">
             <normaloff>:/icons/ic_nel_up_item.png</normaloff>:/icons/ic_nel_up_item.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="downButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="object_viewer.qrc">
             <normaloff>:/icons/ic_nel_down_item.png</normaloff>:/icons/ic_nel_down_item.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Num samples:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="NLQT::CEditRangeUIntWidget" name="nbStepWidget" native="true"/>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CEditRangeUIntWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="object_viewer.qrc"/>
 </resources>
 <connections/>
</ui>
